Tools-Override/tests.targets

```diff
@@ -52,6 +52,8 @@
     <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
 
     <XunitOptions Condition="'$(Performance)'!='true' and '$(Outerloop)' != 'true'">$(XunitOptions) -notrait Benchmark=true</XunitOptions>
+    <!-- Don't run performance tests in parallel, even in "functional" outerloop runs. -->
+    <XunitOptions Condition="'$(Performance)'!='true' and '$(Outerloop)' == 'true' and '$(IncludePerformanceTests)' == 'true'">$(XunitOptions) -parallel none</XunitOptions>
```

Comment:
Right. And we don't need to set this for the "real" performance runner because it already runs things serially.